packages/runtime/src/tutorial-runner.ts

```diff
@@ -484,3 +521,19 @@ async function updateFiles(webcontainer: WebContainer, previousFiles: Files, new
 
   await webcontainer.mount(toFileTree(addedOrModified));
 }
+
+function removeLeadingSlash(filePath: string) {
+  if (filePath.startsWith('/')) {
```

Comment:
This is always true where the function is used. I would probably remove the function and inline the slice.